blingfire-sys/build.rs

```diff
@@ -12,5 +12,7 @@ fn main() {
         "cargo:rustc-link-search=native={}/build",
         destination.display()
     );
-    println!("cargo:rustc-link-lib=blingfiretokdll");
```

Comment:
>  Submodule BlingFire updated from a7b962 to d08f6f 

a7b962 is master, what are you updating to?

Reply:
I think you only have the commits locally, you need to fork the C++ library or submit a PR

Reply:
forked and changed the submodule